packages/stories/src/stories/description.ts

```diff
@@ -0,0 +1,81 @@
+import { bindTemplate, Parameters, TemplateFn } from '@core';
+import { v4 as uuidv4 } from 'uuid';
+
+export interface DescriptionArgs {
+  textBefore: string,
+  textAfter: string,
+  term: string,
+  content: string,
+  open?: boolean,
```

Comment:
Waarom is dit optioneel terwijl in de `.addParameters()` een waarde `false` wordt gezet? Je mag er dan vanuit gaan dat `open` altijd de waarde `false` heeft tenzij in andere stories is aangepast.

Reply:
Zeker, ik ben die helemaal vergeten terug te zetten. Die `open` had ik eerder niet als default meegegeven aan de `.addParameters()`. Ik pas hem aan